pointerevents/extension/pointerevent_constructor.html

```diff
@@ -17,36 +17,79 @@
             var target0 = document.getElementById("target0");
             // set values for non-default constructor
             on_event(target0, "pointermove", this.step_func(function(event) {
-                generate_tests(assert_equals, [
-                    ["getCoalescedEvents().length", event.getCoalescedEvents().length, 2],
-                    ["event.target", event.target, target0],
```

Comment:
The getCoalescedEvents is only in the extension API and is not going to be part of PointerEvent spec v2. So getCoalescedEvents is not event defined in the main folder. For the target the goal was to just have the check for the target of the coalesced events. But I guess it was more of a copy paste that we have the event.target tested here too.